src/SourceBuild/tarball/content/build.sh

```diff
@@ -11,6 +11,7 @@ usage() {
     echo "  --run-smoke-test                   don't build; run smoke tests"
     echo "  --with-packages <dir>              use the specified directory of previously-built packages"
     echo "  --with-sdk <dir>                   use the SDK in the specified directory for bootstrapping"
+    echo "  --use-mono-runtime                 output uses the mono runtime"
```

Comment:
Have you considered something like `--with-runtime=(mono|coreclr)` for the name of this flag?

Reply:
This was the simplest way to expose the feature. Like the repos, the implementation assumes coreclr is the default, and mono the opt-in (with `Conditions`). The argument is exposing that.